members/CusB729dc0101592930443.yaml

```diff
@@ -0,0 +1,119 @@
+bioguide: CusB729dc0101592930443
+contact_form:	
+  method: post	
+  steps:	
+    - visit: "https://eplanning.blm.gov/eplanning-ui/project/68107/595/8001159/comment"	
+    - find: 
+      - selector: "#comment"
+    - fill_in: 
+      - name: Message 
+        selector: "#comment"
+        value: $MESSAGE
+        required: true 
+    - find: 
+      - selector: "#first-name"
```

Comment:
Do these inputs load independently? why do you need all of the `find`s after the initial one for `#comment`?

Reply:
Some of them do, yes. Those are `comment`, `first name` and `last name`. I took the redundant `find` for `first-name` out. 

Other ones like `street` and `city` load after you click on "I would like to provide additional contact information". For these inputs, I'm being a little bit extra careful and adding `find`s to make sure those are handled correctly (this is the first yaml I create for the revamped BLM form).